src/linux/init/init.cpp

```diff
@@ -3169,22 +2986,16 @@ Return Value:
             FATAL_ERROR("Invalid message size {}", Message.size());
         }
 
-        switch (Header->MessageType)
+        if (Header->MessageType == LxInitMessageCreateProcess)
+        {
+            SessionLeaderCreateProcess(Message, MessageFd, std::move(TtyFd), Config);
```

Comment:
We shouldn't move the tty fd here, otherwise the next `LxInitMessageCreateProcess` call wil receive an empty fd.

Given that the fd is owned `SessionLeaderEntry()`, I think passing down the FD by value should be OK here